AppCenterDistribute/AppCenterDistribute/MSACDistribute.m

```diff
@@ -685,7 +685,13 @@ - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret
 
 - (BOOL)openUrlUsingSharedApp:(NSURL *)url {
   UIApplication *sharedApp = [MSACUtility sharedApp];
+#pragma clang diagnostic push
+
+// Ignore "cast to smaller integer type 'BOOL' (aka 'signed char') from 'id'" for Xcode 12.5.
```

Comment:
Warning was correct here. You cannot just ignore it because of "Xcode 12.5".
The good explanation here: https://stackoverflow.com/a/23902487/1703290
If you want to keep this cast, left the comment WHY it works

Reply:
At first I was planning to convert this to NSInvocation. But then tested with muting this error and it also worked.

This method is called when Distribute is trying to call the iOS Tester App for updates : https://github.com/microsoft/appcenter-sdk-apple/blob/develop/AppCenterDistribute/AppCenterDistribute/MSACDistribute.m#L437
So, it is probably not used as there is no ios tester app afaik.

Reply:
Added better description, which is directly based on the answer from stackoverflow.